src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -1366,26 +1365,14 @@ void SolverInterfaceImpl::writeVectorGradientData(
 
     for (int dimSpace = 0; dimSpace < _dimensions; dimSpace++) {
       for (int dimData = 0; dimData < _dimensions; dimData++) {
-
         const int offsetInternal = valueIndex * _dimensions;
+        // Values are entered derived in spatial dimensions first : gradient matrix read rowwise
```

Comment:
What exactly do you mean? From my understanding this would be correct, but I guess the whole 'row/spatial dimension/ components' expression lead in the end only to confusion. Maybe I should just remove the sentence.

Reply:
I got this now. The passive messed this sentence up for me.